src/test/java/com/qaprosoft/carina/demo/mytests/newweb/HeaderTest.java

```diff
@@ -0,0 +1,38 @@
+package com.qaprosoft.carina.demo.mytests.newweb;
+
+import org.testng.asserts.SoftAssert;
+import org.testng.annotations.Test;
+
+import com.qaprosoft.carina.core.foundation.IAbstractTest;
+import com.qaprosoft.carina.demo.gui.components.HeaderMenu;
+import com.qaprosoft.carina.demo.gui.enums.HeaderButtonLink;
+import com.qaprosoft.carina.demo.gui.enums.HeaderIconLink;
+import com.qaprosoft.carina.demo.gui.pages.HomePage;
+import com.zebrunner.agent.core.annotation.TestLabel;
+import com.zebrunner.carina.core.registrar.ownership.MethodOwner;
+
+public class HeaderTest implements IAbstractTest {
+
+    @Test(description = "Verifying all header components are presented")
+    @MethodOwner (owner = "YakubT")
+    @TestLabel(name = "feature", value = {"web", "acceptance"})
+    public void testHeaderComponentsPresent() {
+        HomePage homePage = new HomePage(getDriver());
+        homePage.open();
+        HeaderMenu headerMenu = homePage.getHeaderMenu();
```

Comment:
add assert         Assert.assertTrue(homePage.isPageOpened(), "Home page is not opened");